src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -889,21 +895,29 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
     }
 
     setStoredDonationDonorProperties(dataImport) {
-    if (dataImport[DONATION_DONOR_FIELDS.donationDonorField]) {
+        const donationDonor = dataImport[DONATION_DONOR_FIELDS.donationDonorField];
+        if (donationDonor) {
             this.handleDonationDonorChange(
                 dataImport[DONATION_DONOR_FIELDS.donationDonorField]
             );
         }
         if (dataImport[DATA_IMPORT_ACCOUNT1_IMPORTED_FIELD.fieldApiName]) {
-            this.handleDonorAccountChange(
-                dataImport[DATA_IMPORT_ACCOUNT1_IMPORTED_FIELD.fieldApiName]
-            );
+            this.handleDonorAccountChange(dataImport);
         }
         if (dataImport[DATA_IMPORT_CONTACT1_IMPORTED_FIELD.fieldApiName]) {
             this.handleDonorContactChange(
                 dataImport[DATA_IMPORT_CONTACT1_IMPORTED_FIELD.fieldApiName]
             );
         }
+        if(dataImport[DATA_IMPORT_CONTACT1_LASTNAME_FIELD.fieldApiName]) {
```

Comment:
```suggestion
        if (dataImport[DATA_IMPORT_CONTACT1_LASTNAME_FIELD.fieldApiName]) {
```

Minor thing. Could we add a space after the `if`?